front/src/common/authorization/components/Error403.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import NavBarSNCF from 'common/BootstrapSNCF/NavBarSNCF';
+
+const Error403 = () => {
+  const { t } = useTranslation(['errors', 'common/common']);
+  const navigate = useNavigate();
+  return (
+    <>
+      <NavBarSNCF appName="OSRD" />
+      <main className="mastcontainer mastcontainer-no-mastnav d-flex align-items-center justify-content-center vh-100">
+        <div className="p-3 text-center">
+          <h1>{t('errors:403')}</h1>
```

Comment:
You can remove 'errors:' because it's the default translation file used
```suggestion
          <h1>{t('403')}</h1>
```